lib/gcov/Config.uk

```diff
@@ -0,0 +1,26 @@
+menuconfig LIBGCOV
+	bool "gcov: Code coverage support"
+	default n
+	select VFSCORE
```

Comment:
Hi! This is not only used for binary file output. Is is required in order for the gcov objects to link in all three cases.

Reply:
So if `LIBGCOV_OUTPUT_BINARY_FILE` requires `VFSCORE` and the code that implements binary file output is wrapped around `#if CONFIG_LIBGCOV_OUTPUT_BINARY_FILE`, why is vfscore a linkage requirement for other output types?

Reply:
So I did a quick experiment and from what I see GCC is looking for these as soon as you pass `--coverage`, I guess that's what you meant above. In that case maybe it would be worth to add a comment above these dependencies in  `Config.uk`.